### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Liujingfang1**
This change is to add examples for using apply, status and prune

### CODE REVIEW

**Comment by Liujingfang1** [comment]
@barney-s

File: docs/status.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+```
+cli-utils apply config/hello
+```
+
+Then, get the status of the applied resources
+```
+cli-utils apply status config/hello
+```
+
+The output is like the following
```
Comment (pwittrock) [inline docs/status.md]: what is the exit code if it is done vs not done?

File: docs/prune.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+```
+inventory:
+  type: ConfigMap
+  configMap:
+    name: prune-cm-name
+    namespace: some-namespace
+```
+Then the output of the kustomization directory will contain a ConfigMap resource named `prune-cm-name`, which contains a list of all resources in this kustomization directory. 
+
+When resources are removed from this kustomization directory, the `prune-cm-name` will contains a different list of resources. The `prune` compares the old list with the new list to get a diff and deletes the obsolete resources.
```
Comment (pwittrock) [inline docs/prune.md]: nit: s/will contains/will contain/

File: docs/prune.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# prune
+
+```
+cli-utils prune <dir>
+```
+can prune obsolete resources from a kustomization directory or resources that are with declarative deletion annotation.
+
+## prune kustomize directory
```
Comment (pwittrock) [inline docs/prune.md]: Introduce that there are 2 methods for deleting resources:

- prune by lifecycle annotations (preferred)
  - explicit deletion of resources by annotating them with deletion
  - will not delete generated secrets and configmaps
- prune by inventory configmap
  - implicit deletion of resources by removing the resource from the configuration
  - will delete generated secrets and configmaps

File: docs/status.md (hunk 1 above)
Comment (Liujingfang1) [reply docs/status.md]: The exit code is 0 but the not done case will show the failure.

File: docs/prune.md (hunk 3 above)
Comment (Liujingfang1) [reply docs/prune.md]: prune by inventory configmap doesn't delete generated secrets and configmaps if they are/were refereed in other objects.

File: docs/prune.md (hunk 2 above)
Comment (Liujingfang1) [reply docs/prune.md]: updated

File: docs/prune.md (hunk 3 above)
Comment (Liujingfang1) [reply docs/prune.md]: updated

File: docs/status.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```
+2019/08/07 16:38:04 nil value at `volumeClaimTemplates[].metadata.labels` ignored in mutation attempt
```
Comment (pwittrock) [inline docs/status.md]: what is this?

File: docs/prune.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+The output is like the following
+```
+NAME                                           READY   STATUS    RESTARTS   AGE
+myapp-pod                                      1/1     Running   0         1m
+```
+
+Then run the prune subcommand by
+```
+cli-utils prune config/helloWithInventory
```
Comment (pwittrock) [inline docs/prune.md]: is there a dry-run that prints out what will be deleted?

File: docs/status.md (hunk 4 above)
Comment (Liujingfang1) [reply docs/status.md]: Ah, this is some warning message when applying the example manifests. I can remove it.

File: docs/prune.md (hunk 5 above)
Comment (Liujingfang1) [reply docs/prune.md]: not yet.

**Pull request closed without merging**